app/controllers/submission_controller.rb

```diff
@@ -1,10 +1,11 @@
 class SubmissionController < ApplicationController
   def create
-    @submission = Submission.new(
-      submission_params(params).merge(status: Submission::STATUS[:queued])
+    @submission = Submission.create(
+      submission_params.merge(
+        status: Submission::STATUS[:queued],
```

Comment:
not one for this PR but it maybe that `status` is not being used

Reply:
agreed, good to kill at the same time as submitton_details